waku/v2/peermanager/peer_manager_test.go

```diff
@@ -20,16 +21,22 @@ import (
 
 func getAddr(h host.Host) multiaddr.Multiaddr {
 	id, _ := multiaddr.NewMultiaddr(fmt.Sprintf("/p2p/%s", h.ID().Pretty()))
-	return h.Network().ListenAddresses()[0].Encapsulate(id)
+	var selectedAddr multiaddr.Multiaddr
+	//For now skipping circuit relay addresses as libp2p seems to be returning empty p2p-circuit addresses.
+	for _, addr := range h.Network().ListenAddresses() {
+		if strings.Contains(addr.String(), "p2p-circuit") {
+			continue
+		}
```

Comment:
This change won't work, because the multiaddr returned by libp2p which is not valid is just `/p2p-circuit`. Since this matches with value for protocol, the above check doesn't work. So, keeping the change as is.